docs/guides/backends.rst

```diff
@@ -0,0 +1,199 @@
+.. _backends_guide:
```

Comment:
I think this file should be deleted? We have circuits_runners_guide now?

Reply:
Good catch! I will change the names appropriately.